EnglishFilter/sections/foreign.txt

```diff
@@ -6997,6 +6997,10 @@ forbes.pl##.placeholder
 @@||ocdn.eu/ucs/static/*Piano/*/build/amd/floatingAd.js$domain=forbes.pl
 ! https://github.com/AdguardTeam/AdguardFilters/issues/84630
 meczyki.pl##.vi-ai-banner
+! https://github.com/AdguardTeam/AdguardFilters/issues/86474
+||e-hotelarz.pl/grafika/grafika.php
+e-hotelarz.pl##.pws-reklama
+e-hotelarz.pl###page > .pws-reklama+ br
```

Comment:
`e-hotelarz.pl###page > div.pws-reklama + br`
What about this?
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/124192813-9140e700-dace-11eb-8bf1-31307df7a72e.png)

</details><br/>

Reply:
@zloyden, to make this gap smaller:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/124280575-87ac9300-db51-11eb-9e2f-289a1f5932eb.png)
</details><br/>

should i remove the rule?